app/src/main/java/chat/rocket/android/server/domain/GetChatRoomsInteractor.kt

```diff
@@ -19,4 +20,21 @@ class GetChatRoomsInteractor @Inject constructor(private val repository: ChatRoo
         }
         return chatRooms.await()
     }
+
+    /**
+     * Get a specific room by its id.
+     *
+     * @param serverUrl The server url where the room is.
+     * @param roomId The id of the room to get.
+     *
+     * @return The ChatRoom object or null if we couldn't find any.
+     */
+    suspend fun getById(serverUrl: String, roomId: String): ChatRoom? {
+        return async(CommonPool) {
```

Comment:
return `withContext(CommonPool) { }`  is the same as `return async(CommonPool) { }.await()`